node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -398,6 +452,37 @@ func (o OsDeviceConnectivityHelperGeneric) GetHostsIdByArrayIdentifier(arrayIden
 
 }
 
+func (o OsDeviceConnectivityHelperGeneric) IsMpathHasRightVolumeId(dmPath string,
+	volumeUuidLower string, volumeNguid string) (bool, error) {
+	sgInqWwn, err := o.getSgInqWwnFromDmPath(dmPath)
+	if err != nil {
+		return false, err
+	}
+	if sgInqWwn == volumeUuidLower || sgInqWwn == volumeNguid {
+		return true, nil
+	}
+	return false, &ErrorWrongDeviceFound{dmPath, volumeUuidLower, sgInqWwn}
+}
+
+func (o OsDeviceConnectivityHelperGeneric) IsMpathHasRightVolumeIdWithoutErrors(dmPath string, wnatedVolumeIds []string) bool {
+	sgInqWwn, _ := o.getSgInqWwnFromDmPath(dmPath)
+	for _, wnatedVolumeId := range wnatedVolumeIds {
+		if sgInqWwn == wnatedVolumeId {
+			return true
+		}
+	}
+	return false
+}
+
+func (o OsDeviceConnectivityHelperGeneric) getSgInqWwnFromDmPath(dmPath string) (string, error) {
+	SgInqWwn, err := o.GetWwnByScsiInq(dmPath)
```

Comment:
I don't think local (non-const) variables should start with a capital letter, anywhere
```suggestion
	sgInqWwn, err := o.GetWwnByScsiInq(dmPath)
```

Reply:
done

Reply:
same for internal consts (e.g. `DevMapperPath`)